### PR TITLE
bump k8s libs

### PR DESCRIPTION
**Pull request by xh3b4sd**
Towards https://github.com/giantswarm/giantswarm/issues/2844.

### CODE REVIEW

File: Gopkg.toml
Comment (xh3b4sd) [inline Gopkg.toml]: Waiting for https://github.com/giantswarm/apiextensions/pull/63.

File: Gopkg.toml
Comment (xh3b4sd) [inline Gopkg.toml]: Waiting for https://github.com/giantswarm/operatorkit/pull/176.

**Comment by xh3b4sd** [comment]
Successfully tested on geckon.

**Review by rossf7 — APPROVED** [review]
LGTM once the related PRs are merged.

**Review by kopiczko — APPROVED** [review]

**Review by fgimenez — APPROVED** [review]
LGTM

**Pull request merged**
